Subject: [PATCH 06/24] Fixing corretto-11-bin packaging issue

- do_package expecting libraries at lib folder for packaging,but files are installed in lib64.
- Renaming the lib64 to lib is fixed the issue

---
 .../amazon-corretto/corretto-11-bin_11.0.23.9.1.bb            | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/meta-aws/recipes-devtools/amazon-corretto/corretto-11-bin_11.0.23.9.1.bb b/meta-aws/recipes-devtools/amazon-corretto/corretto-11-bin_11.0.23.9.1.bb
index 8d78f13a..44658f7b 100644
--- a/meta-aws/recipes-devtools/amazon-corretto/corretto-11-bin_11.0.23.9.1.bb
+++ b/meta-aws/recipes-devtools/amazon-corretto/corretto-11-bin_11.0.23.9.1.bb
@@ -43,5 +43,9 @@ RDEPENDS:${PN}-ptest:prepend = "\
     greengrass-bin \
     "

+do_install_append_aarch64() {
+    mv ${D}/usr/lib64 ${D}/usr/lib
+}
+
 # this is used by meta-aws-tests to find this recipe for ptests, so it should stay in this file instead of moving into corretto-bin-common
 inherit ptest
--
2.25.1
